services/proposals/src/propose.py

```diff
@@ -23,21 +24,32 @@ def propose(db_insert_fn: Callable[[Mapping[T, T], T], None]) -> None:
     start_time = time.time()
     client = MongoClient(os.environ["DBCONNECT"])
     logging.info(f'Connected to client: {client}. Setting watch on raw_pdfs collection')
+    pod_name = os.environ["POD_NAME"]
+    podre = re.search(r'\d+$', pod_name)
```

Comment:
Are pod names guaranteed to contain digits if they're part of a replica set?

Reply:
They are guaranteed to contain digits because now they are StatefulSets. They will be in this format for sure.